src/client/api/events/sub_serialized_output.c

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#include "client/api/events/sub_serialized_output.h"
+
+int event_sub_txn_included_msg(event_client_handle_t client, int *mid, char const transaction_id[], int qos) {
```

Comment:
validates the string length of `transaction_id` is 64?

Reply:
Done.